trompace/client/client_itemlist.py

```diff
@@ -0,0 +1,128 @@
+                                        description = description)
+
+    resp = trompace.connection.submit_query(mutation)
+    result = resp.get("data", {}).get("CreateItemList")
+
+    if result:
+        itemlist_id = resp["data"]["CreateItemList"]["identifier"]
+    else:
+        # Raise Error
+        pass
+
+    return itemlist_id
+
+
+def create_listitem_node(contributor = contributor, name = name, 
+                         description = description, position = position):
+    """
+    """
+    mutation = mutation_create_listitem(contributor = contributor, 
+                                        name = name,
+                                        description = value, 
+                                        position = position)
+
+    resp = trompace.connection.submit_query(mutation)
+    result = resp.get("data", {}).get("CreateListItem")
+
+    if result:
+        element_id = resp["data"]["CreateItemList"]["identifier"]
+    else:
+        # Raise Error
+        pass
+
+    return element_id
+
+
+def merge_itemlist_itemlistelement_nodes(itemlist_id = itemlist_id, 
+                                         element_id = element_id):
+    """
+    """
+    mutation = mutation_add_itemlist_itemlist_element(itemlist_id = itemlist_id, 
+                                                      element_id = element_id)
+    resp = trompace.connection.submit_query(mutation)
+    result = resp.get("data", {}).get("MergeItemListItemListElement")
+
+    if not result:
+        # Raise Error
+        pass
+
+
+def merge_listitem_nextitem_nodes(listitem_id = previousItemId, 
+                                  nextitem_id = element_id):
+    """
+    """
+    mutation = mutation_add_listitem_nextitem(listitem_id = previousItemId, 
+                                              nextitem_id = element_id)
+    resp = trompace.connection.submit_query(mutation)
+    result = resp.get("data", {}).get("MergeListItemNextItem")
+
+    if not result:
+        # Raise Error
+        pass
+
+
+def merge_listitem_item_nodes(listitem_id = listitem_id, item_id = item_id):
+    """
+    """
+    mutation =  mutation_add_listitem_item(listitem_id = listitem_id, 
+                                           item_id = item_id)
+
+    resp = trompace.connection.submit_query(mutation)
+    result = resp.get("data", {}).get("MergeListItemItem")
+
+    if not result:
+        # Raise Error
+        pass
+
+
+def create_itemlist(contributor: str, name: str, description: str, 
+                    node_ids: list = None, values: list = None, ordered: bool):
+    """
+    Main function
+    """
+    if values:
+        listitems = values
+    elif node_ids:
+        listitems = node_ids
+
+    itemlist_id = create_itemlist_node(contributor = contributor, name = name, 
+                                       description = description, ordered = ordered):
+
+    position = 0
+    previous_item_id = None
+    for item in listitems:
+        if values:
```

Comment:
This sets the description of the node if you pass in `values`, but it'll be set to the itemlist's description if you pass in nodes. Do something like set listitem_description to None or this value if needed.

Also, I wouldn't check directly against `values` here (and `node_ids` further down). Instead, set a boolean that indicates which mode we're in and check against that. It makes it a bit more explicit as to what the aim is